src/frontend/context/OppgaverContext.tsx

```diff
@@ -280,6 +282,28 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
             });
     };
 
+    interface ITilgangRequestDTO {
+        brukerIdent: string;
+    }
+
+    const sjekkTilgang = async (brukerIdent: string): Promise<ITilgangDTO> => {
+        return axiosRequest<ITilgangDTO, ITilgangRequestDTO>({
+            method: 'POST',
+            url: '/familie-ba-sak/api/tilgang',
+            data: { brukerIdent: brukerIdent },
+        }).then((res: Ressurs<ITilgangDTO>) => {
+            if (res.status !== RessursStatus.SUKSESS) {
+                loggFeil(undefined, undefined, 'RessursStatus er ikke SUKSESS');
+                throw new Error('RessursStatus er ikke SUKSESS');
```

Comment:
Logging av feil håndteres av axiosRequest. Her bør dere eksplisitt håndtere suksess, feil osv.

Reply:
Ok - jeg har pushet en ny variant der `Ressurs` er dyttet til endestasjonene (altså de steder der `sjekkTilgang` brukes). 

`sjekkTilgang` kaller nå kun `axiosRequest`. Konsekvensen er at vi tester for `RessursStatus.SUKSESS` tre steder, men vi unngår å benytte `throw` for å stoppe flyten.

Vi håndterer ikke de andre statusene i `Ressurs` pr. nå.